ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/PortalUserInfo/PortalUserInfo.cpp

```diff
@@ -207,7 +215,9 @@ void PortalUserInfo::onPortalLoadStatusChanged(LoadStatus loadStatus)
     case LoadStatus::Loading:
         break;
     case LoadStatus::FailedToLoad:
-        if (m_portal)
+        if (m_portal->loadError().message() == "Code unauthorized.")
```

Comment:
The previous code had a null check, which makes me wonder if your code still needs one.